pkg/render/fluentd.go

```diff
@@ -121,10 +131,64 @@ type fluentdComponent struct {
 	pullSecrets     []*corev1.Secret
 	installation    *operatorv1.InstallationSpec
 	clusterDomain   string
+	osType          OSType
 }
 
 func (c *fluentdComponent) SupportedOSType() OSType {
-	return OSTypeLinux
+	return c.osType
+}
+
+func (c *fluentdComponent) fluentdName() string {
+	if c.osType == OSTypeWindows {
+		return fluentdWindowsName
+	}
+	return fluentdName
+}
+
+func (c *fluentdComponent) fluentdNodeName() string {
+	if c.osType == OSTypeWindows {
+		return fluentdNodeWindowsName
+	}
+	return fluentdNodeName
+}
+
+func (c *fluentdComponent) eksLogForwarderName() string {
```

Comment:
we don't deploy this for windows right? Do we need to do the name handling for this in that case?

Reply:
The log forwarder should just work on Windows too (the eks forwarder binary is built for windows and added to the fluentd image)